### ISSUE TITLE
Including module references automatically during runtime

### ISSUE THREAD

**Issue by florian-wagner** [sphinx-gallery/sphinx-gallery#52]
I have a python script, which creates the rst files for my module-reference. Now I would like to modify it, such that it automatically includes 

```rst
.. include:: modules/generated/***
.. raw:: html

    <div style='clear:both'></div>
```

where *** should be replaced with the example list generated by sphinxgallery. Yet when I call my script prior to `make html` these files do not exist yet, and if I call it afterwards the module references are not included in the html files. Do you see a clever way to this without building the documentation twice?

Thanks for your help!

**Comment by Titan-C** [comment]
yes, you build documentation it with sphinx autodoc extension an write in the `doc/templates` an rst template with
```
.. include:: {{module}}.{{objname}}.examples
 
.. raw:: html

    <div style='clear:both'></div>
```
You can have a look at how is used in https://github.com/nilearn/nilearn/pull/592/ although that PR needs a rebase. It already solves this issue.

 * [ ] Include this trick in our documentation

**Comment by Titan-C** [comment]
Solved with #63
